text/0002-sarif-result-schema.md

```diff
@@ -0,0 +1,1939 @@
+        "ember/ember-dependencies": 23.327852732,
+        "javascript/eslint-summary": 20.294677398,
+        "ember/ember-test-types": 23.346301169,
+        "ember-octane/ember-octane-migration-status": 8.863200936,
+        "ember/ember-template-lint-disables": 27.640971204,
+        "javascript/eslint-disables": 17.036571088,
+        "javascript/outdated-dependencies": 43.921809849
+      }
+    },
+    "analyzedFiles": [],
+    "analyzedFilesCount": 1539,
+    "actions": [
+      {
+        "name": "reduce-eslint-disable-usages",
+        "summary": "Reduce number of eslint-disable usages",
+        "details": "32 usages of eslint-disable",
+        "defaultThreshold": 2,
+        "items": [
+          "Total eslint-disable usages: 32"
+        ],
+        "input": 32
+      },
+      {
+        "name": "reduce-outdated-major-dependencies",
+        "summary": "Update outdated major versions",
+        "details": "28 major versions outdated",
+        "defaultThreshold": 0.05,
+        "items": [],
+        "input": 0.24347826086956523
+      },
+      {
+        "name": "reduce-outdated-minor-dependencies",
+        "summary": "Update outdated minor versions",
+        "details": "9 minor versions outdated",
+        "defaultThreshold": 0.05,
+        "items": [],
+        "input": 0.0782608695652174
+      },
+      {
+        "name": "reduce-outdated-dependencies",
+        "summary": "Update outdated versions",
+        "details": "32% of versions outdated",
+        "defaultThreshold": 0.2,
+        "items": [],
+        "input": 0.3217391304347826
+      }
+    ]
+  },
+  "runs": [
+    {
+      "results": [
+        {
+          "message": {
+            "text": "components"
+          },
+          "locations": [],
+          "occurrenceCount": 219,
+          "properties": {
+            "taskName": "ember-types",
+            "taskDisplayName": "Ember Types",
+            "category": "metrics",
+            "group": "ember"
+          }
+        },
+        {
+          "message": {
+            "text": "controllers"
+          },
+          "locations": [],
+          "occurrenceCount": 41,
+          "properties": {
+            "taskName": "ember-types",
+            "taskDisplayName": "Ember Types",
+            "category": "metrics",
+            "group": "ember"
+          }
+        },
+        {
+          "message": {
+            "text": "helpers"
+          },
+          "locations": [],
+          "occurrenceCount": 35,
+          "properties": {
+            "taskName": "ember-types",
+            "taskDisplayName": "Ember Types",
+            "category": "metrics",
+            "group": "ember"
+          }
+        },
+        {
+          "message": {
+            "text": "initializers"
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "/app/initializers/app.js"
```

Comment:
Are these paths meant to be absolute? This looks vaguely wrong...

It should be `app/initializers/app.js` no?

It's possible this was my mistake in how the output currently is.

Reply:
Yeah I think there shouldn't  be the initial forward slash there.. I will make sure to fix in the impl, and will correct all paths here